remote_attestation/java/src/com/google/oak/remote_attestation/ClientHandshaker.java

```diff
@@ -91,88 +75,104 @@ public ClientHandshaker(byte[] expectedTeeMeasurement) {
   /**
    * Initializes the remote attestation handshake by creating an {@code ClientHello} message.
    *
-   * Transitions {@code ClientHandshaker} state from {@code Initializing} to
-   * {@code Attesting} state.
+   * Transitions {@code ClientHandshaker} state from {@code State.Initializing} to
+   * {@code State.ExpectingServerIdentity} state.
    */
-  public byte[] createClientHello() throws IllegalStateException, IOException {
-    if (state != HandshakerState.Initializing) {
-      throw new IllegalStateException("ClientHandshaker is not in the Initializing state");
+  public byte[] createClientHello() throws RuntimeException, IOException {
```

Comment:
```suggestion
  public byte[] createClientHello() throws IOException {
```
I am really curious, would not this work?

Reply:
Done